tests/e2e/specs/swap-tokens.spec.js

```diff
@@ -3,48 +3,76 @@ describe('Swap Tokens Tests', () => {
   const limitFloat = float => parseFloat(float.toFixed(5));
   const amountToSwap = 0.001;
   const transactionFee = 0.2;
+  const walletAddress = {
+    value: null,
+  };
   const phrasesList = {
     emerynet: {
       walletButton: 'li[data-value="testnet"]',
       psmNetwork: 'Agoric Emerynet',
       token: 'ToyUSD',
+      isLocal: false,
+      provisionFee: 0.75,
     },
     local: {
       walletButton: 'li[data-value="local"]',
       psmNetwork: 'Local Network',
       token: 'USDC_axl',
+      isLocal: true,
+      provisionFee: 0,
     },
   };
   const networkPhrases = phrasesList[Cypress.env('AGORIC_NET')];
 
   it(`should connect with Agoric Chain on https;//wallet.agoric.app`, () => {
```

Comment:
I don't think we need to visit wallet.agoric.app at all anymore, this whole test case should be removed

Reply:
this code is here to get the address of the wallet, which we use to get IST from the faucet.

Reply:
Can you get the address from keplr instead? Or if the seed phrase is predetermined, the address should be deducible as well, no need to query it.

Reply:
Ah I think it's because the account doesn't have any tokens yet, so Keplr doesn't try to show you emerynet there. You'd have to go into "Manage Chain Visibility" and check off emerynet to see the address. However, an easier thing to do is when setting up the wallet, check off "Agoric":

![image](https://github.com/Agoric/dapp-psm/assets/8848650/fb22da0a-c8d2-406b-94da-eb821b5283cd)

The address would be the same between emerynet and Agoric mainnet. It's really not ideal to depend on the live service of wallet.agoric.app in this test because it's not part of the behavior being tested and that app breaking should not impact the psm. If this change needs to be made in @agoric/synpress I think it's worth doing now because it will come up again.

Reply:
Thank you for the amazing idea! works like a charm. updated accordingly